angular.json

```diff
@@ -663,7 +663,253 @@
           }
         }
       }
+    },
+    "storefrontapp-gigya-integration": {
```

Comment:
We recently switched to having only one testing application (to make maintenance easier).
To enable different features we use `environment.ts` and `environment.prod.ts` and dedicated file with configuration for each feature. You can watch at `cds` as it has quite similar setup to this feature.

Could you remove the extra testing app and integrate this feature in storefrontapp?

Reply:
I thought about giving you this early feedback, so you have something to work on until I finish review of the rest.

Reply:
Removed the sample app and added logic to use storefrontapp itself as the testing application.